src/similarity.rs

```diff
@@ -79,24 +79,29 @@ pub fn pairwise_entity_resnik_score(
 }
 
 pub fn calculate_max_information_content(
```

Comment:
Because this takes 5 args now, running test_calculate_max_information_content throws an error:
```
error[E0061]: this function takes 5 arguments but 4 arguments were supplied
   --> src/similarity.rs:424:22
    |
424 |         let result = calculate_max_information_content(
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
425 |             &closure_table,
426 |             &String::from("CARO:0000000"),
    |             ----------------------------- an argument of type `&std::collections::HashMap<String, f64>` is missing
```

Reply:
yep, thanks - at least one test is broken (and others yet to be written)